RELEASE.md

```diff
@@ -0,0 +1,253 @@
+# Release Process
+
+## Introduction
+
+The process for managing enhancements (OKEPs), issues, and pull requests through an OVN-Kubernetes release is complex. This document describes the process in detail.
+
+## How often are releases?
+
+We release approximately once every 3 calendar months.
+
+## How do we decide on the release date?
+
+The release date is decided during the [community meeting] at the beginning of the release cycle.
+It will be approximately 3 calendar months from the previous release. The release date will not land on a Friday, holiday, or weekend.
+
+## Who is responsible for the release?
+
+The release coordinator is elected at the [community meeting] at the beginning of the release cycle.
+The release coordinator may be any volunteer from the community - a call for volunteers will be issued one week before the next community meeting via the community Slack and mailing list.
+
+### Release Coordinator Responsibilities
+
+- The release coordinator is the person who is ultimately responsible for ensuring that the release is completed on time, and with the desired payload.
+- The release coordinator follows the release process outlined in [#what-is-the-release-process].
+- The release coordinator ensures that all community members are following the correct process.
+- While not a maintainer, the release coordinator is an important role in the community and should be afforded the respect and trust of the maintainers.
+- The release coordinator is empowered to make decisions about the release that are in the best interest of the community.
+  - For example, the release coordinator may choose to delay the release if the payload is not ready or if the quality of the release is not high enough.
+- The release coordinator is responsible for ensuring that progress is being made on the release and regularly updates the community on the progress of the release during the [community meeting].
+- If decisions are required that are outside the scope of the release coordinator's responsibilities, the release coordinator will seek the advice of the maintainers.
+- If the release coordinator is unable to fulfill their responsibilities, they may step down and a new release coordinator will be elected.
+
+## What goes into a release?
+
+A release is a collection of changes that are considered stable and ready for production use.
+These changes can be broken down into the following categories:
+
+- **New features** - new features require an approved OKEP.
+- **Minor changes** - small changes to improve the project.
+- **Bug fixes** - fixes for bugs.
+- **Documentation** - improvements to the documentation.
+- **Testing** - improvements to the testing or CI workflows.
```

Comment:
how are these selected ? 

In my past experience in this community, any minor change, documentation, or testing improvement have low odds of getting attention / traction. Would it make sense to try to define a process for getting some predictability into this type of work ?

Reply:
yes it would.
but i think we need to be mindful of how much process we add at once.
maybe start with the bare minimum, and then if we feel that work isn't getting prioritized, then adjust accordingly?